src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs

```diff
@@ -21,57 +21,65 @@ public KestrelTrace(ILogger logger)
         }
 
         public void ConnectionStop(long connectionId)
         {
-            _logger.LogDebug(14, $"{nameof(ConnectionStop)} -> Id: {connectionId}");
+            _logger.LogDebug(2, @"Connection id ""{ConnectionId}"" stopped.", connectionId);
         }
 
-        public void ConnectionRead(long connectionId, int status)
+        public void ConnectionRead(long connectionId, int count)
         {
-            _logger.LogDebug(4, $"{nameof(ConnectionRead)} -> Id: {connectionId}, Status: {status}");
+            // Don't log for now since this could be *too* verbose.
+            //_logger.LogVerbose(3, @"Connection id ""{ConnectionId}"" received ""{Count}"" bytes.", connectionId, count);
         }
 
         public void ConnectionPause(long connectionId)
         {
-            _logger.LogDebug(5, $"{nameof(ConnectionPause)} -> Id: {connectionId}");
+            _logger.LogDebug(4, @"Connection id ""{ConnectionId}"" paused.", connectionId);
         }
 
         public void ConnectionResume(long connectionId)
         {
-            _logger.LogDebug(6, $"{nameof(ConnectionResume)} -> Id: {connectionId}");
+            _logger.LogDebug(5, @"Connection id ""{ConnectionId}"" resumed.", connectionId);
         }
 
         public void ConnectionReadFin(long connectionId)
         {
-            _logger.LogDebug(7, $"{nameof(ConnectionReadFin)} -> Id: {connectionId}");
+            _logger.LogDebug(6, @"Connection id ""{ConnectionId}"" received FIN.", connectionId);
         }
 
-        public void ConnectionWriteFin(long connectionId, int step)
+        public void ConnectionWriteFin(long connectionId)
         {
-            _logger.LogDebug(8, $"{nameof(ConnectionWriteFin)} -> Id: {connectionId}, Step: {step}");
+            _logger.LogDebug(7, @"Connection id ""{ConnectionId}"" sending FIN.");
+        }
+
+        public void ConnectionWroteFin(long connectionId, int status)
+        {
+            _logger.LogDebug(8, @"Connection id ""{ConnectionId}"" sent FIN with status ""{Status}"".", status);
         }
 
         public void ConnectionKeepAlive(long connectionId)
         {
-            _logger.LogDebug(9, $"{nameof(ConnectionKeepAlive)} -> Id: {connectionId}");
+            _logger.LogDebug(9, @"Connection id ""{ConnectionId}"" completed keep alive response.", connectionId);
         }
 
         public void ConnectionDisconnect(long connectionId)
         {
-            _logger.LogDebug(10, $"{nameof(ConnectionDisconnect)} -> Id: {connectionId}");
+            _logger.LogDebug(10, @"Connection id ""{ConnectionId}"" disconnected.", connectionId);
         }
 
         public void ConnectionWrite(long connectionId, int count)
         {
-            _logger.LogDebug(11, $"{nameof(ConnectionWrite)} -> Id: {connectionId}, Count: {count}");
+            // Don't log for now since this could be *too* verbose.
+            //_logger.LogVerbose(11, @"Connection id ""{ConnectionId}"" sent ""{Count}"" bytes.", connectionId, count);
```

Comment:
@davidfowl This was to make it a little easier to do some debugging when working on Kestrel itself. I discussed this with @lodejard and he was fine with it. Do you feel strongly about this?